common/src/main/java/net/william278/husksync/database/Database.java

```diff
@@ -254,15 +254,18 @@ public final void pinSnapshot(@NotNull User user, @NotNull UUID versionUuid) {
      * Identifies types of databases
      */
     public enum Type {
```

Comment:
You could use lombok `@AllArgsConstructor` and `@Getter`

Reply:
this is deprecated, removing

Reply:
I think you can still use the lombok constructor.